pkg/sysregistries/system_registries.go

```diff
@@ -0,0 +1,84 @@
+package sysregistries
+
+import (
+	"github.com/BurntSushi/toml"
+	"io/ioutil"
+)
+
+//systemRegistriesConfPath is the path to the system-wide registry configuration file
+//and is used to add/subtract potential registries for obtaining images.
+//You can override this at build time with
+//-ldflags '-X github.com/containers/image/sysregistries.systemRegistriesConfPath=$your_path'
+var systemRegistriesConfPath = builtinRegistriesConfPath
+
+//builtinRegistriesConfPath is the path to registry configuration file
+//DO NOT change this, instead see systemRegistriesConfPath above.
+const builtinRegistriesConfPath = "/etc/containers/registries.conf"
+
+type registries struct {
+	Registries []string `toml:"registries"`
+}
+
+type tomlConfig struct {
+	Registries struct {
+		Search   registries `toml:"search"`
+		Insecure registries `toml:"insecure"`
+		Block    registries `toml:"block"`
+	} `toml:"registries"`
+}
+
+//Reads the global registry file from the filesystem. Returns
+//a byte array
+func readRegistryConf() ([]byte, error) {
+	configBytes, err := ioutil.ReadFile(systemRegistriesConfPath)
+	if err != nil {
+		return nil, err
+	}
+	return configBytes, nil
+}
+
+// For mocking in unittests
+var readConf = readRegistryConf
```

Comment:
Totally non-blocking: A global state like this is a bit awkward, e.g. for running tests in parallel.

Could this instead be done by having `loadRegistryConfWithReader`, which is tested, and an untested `loadRegistryConf() { return loadRegistryConfWithReader(readConf) }` ?